### PR TITLE
feat: update profile menu

### LINKED ISSUE #2085: After adding new ORG-> click 'view my profile' -> unable to edit the profile -> because the selected profile is still individual user profile.
1. login using valid email/pwd
2. Click on profile -> add new ORG
3. Enter the required details on 'Create ORG profile' page -> click 'Create profile'
4. after the profile created on the next page, click 'View my profile'

Issue: You will observe that the profile page of the newly created ORG is displayed, as expected.
BUT, user cannot EDIT it because the selected profile is the 'Individual' user profile.

So, either we have to change the message of 'View "my" profile' with 'View "Organisation" profile' so that user should know, that they are in their ORG profile so that's why no EDIT rights, unless they switch account to ORG account.

OR we can switch the selected profile to the newly created ORG profile, so that user should be able to edit the profile after creation .

Refer below GIF.
ezgif com-video-to-gif

### PR DESCRIPTION
**Pull request by OzkanAbdullahoglu**
* After new organisation created when the user clicks to go to my profile,
* We switch to the newly created organisation.
Resolves #2085 

### What does this pull request aim to achieve? 💯

_Please be concise and link any related issue(s):_

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [x] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

**Comment by OzkanAbdullahoglu** [comment]
@rabbi08 Can you re-test in this branch? I was kinda made this faster than normal and did not very well test.

**Comment by rabbi08** [comment]
@OzkanAbdullahoglu 
I have tested the fix on the feature branch, and it is working fine: PASS
Please refer below gif. Thanks

ezgif com-video-to-gif-4

**Review by BMariscal — DISMISSED** [review]

**Comment by Zaydme** [comment]
Would it be possible if we just update the organizationId in localStorage and then redirect, this way we will not need all the isNewOrganization logic. But if that was causing any issues, this is good as well.

**Comment by OzkanAbdullahoglu** [comment]
> Would it be possible if we just update the organizationId in localStorage and then redirect, this way we will not need all the isNewOrganization logic. But if that was causing any issues, this is good as well.

Usually, I am not a fan of localStorage, It is slower. Because of that, I used redux state. Is it worth to try?

**Comment by Zaydme** [comment]
I mean since we use localStorage already for persistency, you will have to update the localStorage anyway in case the user re-opened the website later, they must be on the same profile.
And localStorage isn't really slow, this is client side at the end ¯\_(ツ)_/¯

**Comment by OzkanAbdullahoglu** [comment]
> I mean since we use localStorage already for persistency, you will have to update the localStorage anyway in case the user re-opened the website later, they must be on the same profile.
> And localStorage isn't really slow, this is client side at the end ¯_(ツ)_/¯

I believe we should not use localStorage for persistence either. (This should be one of refactoring case for the near future). I will check the consistency.

**Comment by OzkanAbdullahoglu** [comment]
@Zaydme you can make your review with this new logic.

File: client/src/pages/ProfileCompleted.js [hunk 1]
```diff
@@ -16,13 +17,22 @@ import {
...
   const { t } = useTranslation();
   const location = useLocation();
   useEffect(() => {
     refetchUser();
     // will only run once (on mount) to update navbar org list
   }, [refetchUser]);
+
+  const switchAccountDrawerMenu = () => {
+    if (location?.state?.orgId) {
+      const index = location.state.orgId;
```
Comment (Zaydme) [inline client/src/pages/ProfileCompleted.js:30]: I am only not sure about this part.
Does the router state have the newly created orgId?

File: client/src/reducers/session/reducer.js [hunk 2]
```diff
@@ -13,6 +13,7 @@ const initialState = {
   forgotPasswordRequested: false,
   user: null,
   organisationId: null,
+  isNewOrganisation: false,
```
Comment (Zaydme) [inline client/src/reducers/session/reducer.js]: Do we still need this?

File: client/src/templates/layouts/NavigationLayout.js [hunk 3]
```diff
@@ -72,6 +72,7 @@ const NavigationLayout = (props) => {
     webSocket,
     loggedInOnly,
     organisationId,
+    isNewOrganisation,
```
Comment (Zaydme) [inline client/src/templates/layouts/NavigationLayout.js]: Same as above

File: client/src/reducers/session/reducer.js (hunk 2 above)
Comment (OzkanAbdullahoglu) [reply client/src/reducers/session/reducer.js]: I forgot this one and the other. of course not.

File: client/src/pages/ProfileCompleted.js (hunk 1 above)
Comment (OzkanAbdullahoglu) [reply client/src/pages/ProfileCompleted.js:30]: Yeap, below sample from `CreateOrganisationProfile` 

code

**Review by Zaydme — APPROVED** [review]
LGTM!

**Pull request merged**